features/recommendations.feature

```diff
@@ -50,4 +50,9 @@ Scenario: Read recommendation with query parameter type
 Scenario: Read recommendation with query parameter product-id
 	When I click "/recommendations" with product-id "2"
 	Then I should see "up-sell"
-	And I should not see "1"
\ No newline at end of file
+	And I should not see "1"
+
+Scenario: Update a recommendation
+	When I update "/recommendations" with id "2" and parent product "1", related product "3", type "up-sell", and priority "8"
+	Then I should see "8"
```

Comment:
I feel like there should be a step to visit the homepage to verify that the update is persisted.